coordinator/src/main/scala/filodb.coordinator/queryplanner/LogicalPlanParser.scala

```diff
@@ -209,6 +209,12 @@ object LogicalPlanParser {
     }$ClosingCurlyBraces"
   }
 
+  private def applyLimitToQuery(lp: ApplyLimitFunction): String = {
+    val periodicSeriesQuery = convertToQuery(lp.vectors)
+    s"$periodicSeriesQuery limit ${lp.limit}"
```

Comment:
Just curious if `limit 1 ` will return 1 time series or 2 time series at the end if we have two partition?